.github/workflows/review.yaml

```diff
@@ -102,7 +102,7 @@ jobs:
             requirements-dev.txt
```

Comment:
Is this prior line going to be affected by the dynamic generation from `install-deps`?

Reply:
Technically it will, but that's frankly very hard to actually program.
The cache will still be useful -- it'll get read based on the two main files, and the downloaded packages from the install step will get written back, but it will take an efficiency hit when we change a dependency in a file not listed here.

Might be worth an issue?